doc/customizing_guide/mpl_viewer/config.py

```diff
@@ -76,6 +76,7 @@ def _on_visual_change(self, value=None):
         else:
             self.artist.set_markerfacecolor('white')
         self.artist.set_alpha(self.state.alpha)
+        self.artist.set_preferred_cmap(self.state.preferred_cmap)
```

Comment:
I don't think you need to include this, it probably won't be relevant for many viewers

Reply:
I tried running glue with L79 commented out, but got the error:
```
Failed to add data
'ImageLayerState' object has no attribute 'preferred_cmap'
```
So maybe we should keep it?

Reply:
The error shows up when trying to view the loaded `SunPy Map` object, BTW...

Reply:
@kakirastern - I think this indicates that the layer state needs to have a default value set to None. I think it would still be best to not show this in the ``config.py`` file here, and it's a good test that it should be possible to not set the preferred cmap.

Reply:
@astrofrog Thanks for the tip! Let me try your suggestion and make changes soon.

Reply:
Done, it worked! 👍